app/assets/src/components/visualizations/TidyTree.js

```diff
@@ -14,8 +14,10 @@ export default class TidyTree {
         addOverlays: true,
         attribute: "aggregatescore",
         leafNodeHeight: 35,
-        minWidth: 1000,
-        minHeight: 300,
+        width: 1000,
```

Comment:
Do not set width and height directly. Just adjust to parent div, so users can set the size of the container as desired (css or js)

Reply:
Or at list make the defaults match the div below...